lib/repositories/genre/Impl/genre_from_json.dart

```diff
@@ -0,0 +1,35 @@
+import 'dart:convert';
+import 'package:flutter/services.dart';
+
+import '../i_genre_repository.dart';
+
+class GenresFromJson extends GenresRepository {
+  late final Map<int, String> genresById;
+  static const String genresPath = 'jsonData/genres.json';
+
+  GenresFromJson() {
+    genresById = {};
+  }
+
+  @override
+  Future<String?> getGenre(int id) async {
+    if (genresById.isEmpty) {
+      genresById.addAll(await parseJsonGenres());
+    }
+    return genresById[id];
+  }
+
+  static Future<Map<int, String>> parseJsonGenres() async {
+    var jsonPath = await rootBundle.loadString('jsonData/genres.json');
```

Comment:
Here you are using an String, that you define as a static const, you can replace this with **genresPath**

Reply:
Solved, I made the const and forgot to use it😅